### ISSUE TITLE
Fuannotate comapre Aborted

### ISSUE THREAD

**Issue by SolayMane** [nextgenusfs/funannotate#953]
**Are you using the latest release?**
Running 1.8.15
**Describe the bug**

**What command did you issue?**
funannotate compare -i GCA_000149955.2_ASM14995v2_genomic/annotate_results/Fusarium_oxysporum_f._sp._lycopersici_4287_4287.gbk GCA_000260155.3_FO_CL57_V1_genomic/annotate_results/Fusarium_oxysporum_f._sp._radicis-lycopersici_26381_26381.gbk --cpus 16 -o xx

**Logfiles**
[Aug 24 05:48 PM]: OS: Ubuntu 14.04, 56 cores, ~ 396 GB RAM. Python: 3.8.15
[Aug 24 05:48 PM]: Running 1.8.15
[Aug 24 05:48 PM]: Now parsing 2 genomes
[Aug 24 05:48 PM]: working on Fusarium oxysporum f. sp. lycopersici 4287
[Aug 24 05:48 PM]: working on Fusarium oxysporum f. sp. radicis-lycopersici 26381
[Aug 24 05:49 PM]: No secondary metabolite annotations found
[Aug 24 05:49 PM]: Summarizing PFAM domain results
/home/inra/miniconda2/envs/funannotate/lib/python3.8/site-packages/sklearn/manifold/_mds.py:298: FutureWarning: The default value of `normalized_stress` will change to `'auto'` in version 1.4. To suppress this warning, manually set the value of `normalized_stress`.
  warnings.warn(
[Aug 24 05:49 PM]: Summarizing InterProScan results
[Aug 24 05:49 PM]: Loading InterPro descriptions
[Aug 24 05:49 PM]: Summarizing MEROPS protease results
[Aug 24 05:49 PM]: found 14/94 MEROPS familes with stdev >= 1.000000
[Aug 24 05:49 PM]: Summarizing CAZyme results
[Aug 24 05:49 PM]: found 30/148 CAZy familes with stdev >= 1.000000
[Aug 24 05:49 PM]: Summarizing COG results
[Aug 24 05:49 PM]: Summarizing secreted protein results
[Aug 24 05:49 PM]: Summarizing fungal transcription factors
[Aug 24 05:49 PM]: No transcription factor IPR domains found
[Aug 24 05:49 PM]: Running GO enrichment for each genome
*** Error in `/home/inra/miniconda2/envs/funannotate/bin/python': double free or corruption (!prev): 0x000055be66dd7de0 ***
Aborted

**OS/Install Information**
-------------------------------------------------------
Checking dependencies for 1.8.15
-------------------------------------------------------
You are running Python v 3.8.15. Now checking python packages...
biopython: 1.76
goatools: 1.2.3
matplotlib: 3.7.2
natsort: 8.3.1
numpy: 1.24.3
pandas: 1.5.3
psutil: 5.7.0
requests: 2.31.0
scikit-learn: 1.3.0
scipy: 1.10.1
seaborn: 0.12.2
All 11 python packages installed

You are running Perl v b'5.032001'. Now checking perl modules...
Carp: 1.50
Clone: 0.46
DBD::SQLite: 1.72
DBD::mysql: 4.046
DBI: 1.643
DB_File: 1.858
Data::Dumper: 2.183
File::Basename: 2.85
File::Which: 1.24
Getopt::Long: 2.54
Hash::Merge: 0.302
JSON: 4.10
LWP::UserAgent: 6.67
Logger::Simple: 2.0
POSIX: 1.94
Parallel::ForkManager: 2.02
Pod::Usage: 1.69
Scalar::Util::Numeric: 0.40
Storable: 3.15
Text::Soundex: 3.05
Thread::Queue: 3.14
Tie::File: 1.06
URI::Escape: 5.17
YAML: 1.30
local::lib: 2.000029
threads: 2.25
threads::shared: 1.61
All 27 Perl modules installed

Checking Environmental Variables...
$FUNANNOTATE_DB=/your/path
$PASAHOME=/home/inra/miniconda2/envs/funannotate/opt/pasa-2.5.2
$TRINITY_HOME=/home/inra/miniconda2/envs/funannotate/opt/trinity-2.8.5
$EVM_HOME=/home/inra/miniconda2/envs/funannotate/opt/evidencemodeler-1.1.1
$AUGUSTUS_CONFIG_PATH=/home/inra/miniconda2/envs/funannotate/config/
        ERROR: GENEMARK_PATH not set. export GENEMARK_PATH=/path/to/dir
-------------------------------------------------------
Checking external dependencies...
/home/inra/miniconda2/envs/funannotate/bin/emapper.py:4: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  __import__('pkg_resources').run_script('eggnog-mapper==2.1.12', 'emapper.py')
PASA: 2.5.2
CodingQuarry: 2.0
Trinity: 2.8.5
augustus: 3.5.0
bamtools: bamtools 2.5.1
bedtools: bedtools v2.31.0
blat: BLAT v37x1
diamond: 2.0.11
emapper.py: 2.1.12
ete3: 3.1.2
exonerate: exonerate 2.4.0
fasta: 36.3.8g
glimmerhmm: 3.0.4
gmap: 2023-04-28
hisat2: 2.2.1
hmmscan: HMMER 3.1b2 (February 2015)
hmmsearch: HMMER 3.1b2 (February 2015)
java: 11.0.13
kallisto: 0.46.1
mafft: v7.520 (2023/Mar/22)
makeblastdb: makeblastdb 2.14.0+
minimap2: 2.26-r1175
pigz: 2.6
proteinortho: 6.2.3
pslCDnaFilter: no way to determine
salmon: salmon 0.14.1
samtools: samtools 1.16.1
signalp: 5.0b
snap: 2006-07-28
stringtie: 2.2.1
tRNAscan-SE: 2.0.11 (Oct 2022)
tantan: tantan 40
tbl2asn: 25.8
tblastn: tblastn 2.14.0+
trimal: trimAl v1.4.rev15 build[2013-12-17]
trimmomatic: 0.39
        ERROR: gmes_petap.pl not installed

**Comment by hyphaltip** [comment]
if you want to upload those GBK files as an attachment we can test it locally - it sounds like a python/GO error in a dependency?  

BTW @nextgenusfs - I kind of think the comparative framework might be better to move to an independent package though I know the parsing re-using the running steps overlap.

**Comment by SolayMane** [comment]
Here are the gbk files : [file1](https://drive.google.com/file/d/1ENFdjUMk2BJmQjKNApUQBg1l99FCvvep/view?usp=sharing) and [file2]( https://drive.google.com/file/d/1JC_IMlJf208DrfZWOBvjIGiomuAv3Guh/view?usp=sharing)

**Comment by nextgenusfs** [comment]
Yes, compare should probably be listed as not supported. It will be dropped in funannotate2. It was written for my use case years ago and probably isn't as useful anymore. I've not seen that error before with go enrichment.
